test/Libraries/PackageManagerTests/PackageLoaderTests.cs

```diff
@@ -657,6 +646,92 @@ public void HasValidStandardLibraryPath()
             Assert.IsNotNullOrEmpty(standardDirectory);
             Assert.AreEqual(standardDirectory, directory);
         }
+        [Test]
+        public void PackageInStandardLibLocationIsLoaded()
+        {
+           //setup clean loader
+            var loader = new PackageLoader(new string[0], StandardLibraryTestDirectory);
+            var settings = new PreferenceSettings();
+            settings.DisableStandardLibrary = false;
+
+            var loaderParams = new LoadPackageParams()
+            {
+                PathManager = CurrentDynamoModel.PathManager,
+                Preferences = settings
+            };
+            //invoke the load
+            loader.LoadAll(loaderParams);
+
+            //assert the package in std lib was not loaded.
+            Assert.AreEqual(1, loader.LocalPackages.Count());
+            Assert.IsTrue(loader.LocalPackages.Any(x => x.BinaryDirectory.Contains("SignedPackage2")));
+        }
+
+        [Test]
+        public void DisablingStandardLibraryCorrectlyDisablesLoading()
+        {
+
+            //setup clean loader
+            var loader = new PackageLoader(new string[0], StandardLibraryTestDirectory);
+            var settings = new PreferenceSettings();
+            settings.DisableStandardLibrary = true;
+
+            var loaderParams = new LoadPackageParams()
+            { PathManager = CurrentDynamoModel.PathManager,
+                Preferences = settings };
+            //then invoke load
+
+            loader.LoadAll(loaderParams);
+
+            //assert the package in std lib was not loaded.
+            Assert.AreEqual(0, loader.LocalPackages.Count());
+            Assert.IsFalse(loader.LocalPackages.Any(x => x.Name.Contains("SignedPackage2")));
```

Comment:
Is this assertion really necessary since we have already asserted that the count is zero?

Reply:
no, these assertions are not necessary, but I can either move them up a line, or just turn this into a comment - I thought it would be useful to encode which package the test is expected to work with incase someone adds another package to that location.